README.md

```diff
@@ -400,9 +400,62 @@ See [the postgres documentation about SSL](https://www.postgresql.org/docs/11/li
 
 See [the postgres documentation about encoding](https://www.postgresql.org/docs/11/multibyte.html) for more information.
 
+### SSL Client connection using default certificates
 To force SSL connection between clients you need to use the environment 
 variable `FORCE_SSL=TRUE`
 
+If you are using the default certificates provided by the image when connecting
+to the database you will need to set `SSL Mode` to any value besides
+`verify-full` or `verify-ca`
+
+### SSL Client connection using user defined certificates
+To force SSL connection between clients you need to use the environment 
+variable `FORCE_SSL=TRUE`
+
+If you are using your own certificates when connecting to the database you will need to set
+`SSL Mode` to force the connection to use SSL.
+
+You also need to define the following environment variables when setting up the database connection.
+
+SSL_CERT_FILE:
+
+SSL_KEY_FILE: 
+
+SSL_CA_FILE:
+
+On the host machine where you need to connect to the database you also 
+need to copy the `SSL_CA_FILE` file to the location `/home/$user/.postgresql/root.crt`
+or define an environment variable pointing to location of the `SSL_CA_FILE`
+example: `PGSSLROOTCERT=/etc/letsencrypt/root.crt`
+
+#### SSL connection inside the docker container using openssl certificates
+
+
+Generate the certificates inside the container
+```
+LETSENCRYPT_CERT_DIR=/etc/letsencrypt
```

Comment:
Does this make sense? We are not actually using letsencrypt in this case right?

Reply:
This is just an example of how you could use the option you mentioned in the comment `Forced SSL with Certificate Exchange`

Reply:
I will change the directory name to be generic

Reply:
Thanks